dumb.coffee.js

```diff
@@ -0,0 +1,3 @@
+class this_isClearly_Wrong
```

Comment:
Identifier 'this_isClearly_Wrong' is not in camel case.<br>'class' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).